src/components/Header.js

```diff
@@ -0,0 +1,20 @@
+const Header = ({ setItemSearch, setCount }) => {
```

Comment:
Please, move form to the separate component

Reply:
done